src/codegen/query_compiler.cpp

```diff
@@ -40,20 +40,25 @@ std::unique_ptr<Query> QueryCompiler::Compile(
   return query;
 }
 
-bool QueryCompiler::IsSupported(const planner::AbstractPlan &plan) {
-  return QueryCompiler::IsSupported(plan, nullptr);
+bool QueryCompiler::IsSupported(UNUSED_ATTRIBUTE const planner::AbstractPlan &plan) {
+  //return QueryCompiler::IsSupported(plan, nullptr);
+  return false;
```

Comment:
This is mainly due to https://github.com/cmu-db/peloton/issues/654. We will delete this one later.